asciidoctorj-distribution/src/test/groovy/org/asciidoctor/diagram/WhenDitaaDiagramIsRendered.groovy

```diff
@@ -27,8 +27,7 @@ class WhenDitaaDiagramIsRendered extends Specification {
 
         given:
         String imageFileName = UUID.randomUUID()
-        File imagesOutDir = testFolder.newFolder()
-        File createdImage = new File(imagesOutDir, "${imageFileName}.png")
+        File imagesOutDir = new File(testFolder.root, "images-dir")
```

Comment:
I was checking right now. Really this is my worst PR in years XD but I take it with humor.